### DISCUSSION TITLE
My Nordic Didn't Connect to Artisan v2.4.6

### DISCUSSION THREAD

**Discussion by Gochujunk** [artisan-roaster-scope/artisan#899] [Q&A]
Hello Guys,

Kindly need your advice due to Artisan v2.4.6 didn't want to read my Nordic Machine.
But one of my friend using the V1.0 there's no problem with it.

is there any advice? or can i have the older version like v.1.0 by any chance?

**Comment by MAKOMO** [comment]
You should always use the latest version of Artisan, which is currently v2.6.0. There is no advantage of the v1.0 vs newer Artisan versions. However, the newer Artisan versions perform way better than the outdated v1.0.  

There are 4 different settings for Nordic machines supported by Artisan v2.6.0. Do you know which type of machine you have? Did you test all 4 settings? Did you have the PL2303 serial driver installed and the correct serial port selected, in case your machine connect via serial/USB connection? In case your machine features a networked PLC, did you have the correct IP setup to connect to the machine?

**Reply by Gochujunk** [reply]
So i have the Nordic R1 first generation 2019, and i bought it from my friend and he’s using the Artisan V1.0.0 and it works for him. For the settings problem it doesn’t work at all when i test all 4 settings that available.

**Reply by MAKOMO** [reply]
Did you install the PL2303 serial driver? Is your machine connected via USB/Serial or Network? Did you try to use your friends setup (export settings via Help menu on his Artisan and import it into yours)? Did you ask Nordic for support, I guess they know their machines best?

For sure the Artisan version is not the problem.
